package.json

```diff
@@ -18,20 +18,20 @@
   ],
   "scripts": {
     "start": "node src/index.js",
+    "dev": "nodemon src/index.js",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
-  "engines": {
-    "node": ">=4.2.0",
-    "npm": ">=2.14.7"
-  },
   "dependencies": {
     "axios": "^0.19.0",
     "body-parser": "^1.19.0",
     "debug": "^4.1.1",
     "dotenv": "^6.2.0",
     "express": "^4.17.1",
-    "level": "^4.0.0",
+    "level": "^6.0.0",
```

Comment:
Use the older version to make the Level DB work as-is. Or the code needs to be re-written. Since this is not the most important part, I think keeping it old is okay but up to you!